components/brain/content/BrainContentPinnedWaves.tsx

```
@@ -4,11 +4,22 @@ import React, { useEffect, useRef, useState } from "react";
+import useDeviceInfo from "@/hooks/useDeviceInfo";
+import { getWaveHomeRoute } from "@/helpers/navigation.helpers";
+import { useMyStream } from "@/contexts/wave/MyStreamContext";
 
 const BrainContentPinnedWaves: React.FC = () => {
   const router = useRouter();
   const searchParams = useSearchParams();
   const { pinnedIds, addId, removeId } = usePinnedWaves();
+  const { isApp } = useDeviceInfo();
+  let directMessagesList: ReadonlyArray<{ id: string }> = [];
+  try {
+    const stream = useMyStream();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)